### PR TITLE
Add Aither

### PR DESCRIPTION
**Pull request by Triharder-E7**
Here's Aither's hero page with the new skill-enhancement format and all that good stuff! I updated a few small errors in the hero's Readme file as well. I wanted to do a single hero before submitting batches, in case I'm doing something silly, thanks! :)

### CODE REVIEW

**Comment by kmalone86** [comment]
Meant to do a proposed change, but guess I did that wrong. Sorry about that. I am not sure why adding in 4 spaces would cause the checker to fail.

All the data looks good, just seeing some spacing oddities. I see a few tabs in files changed in enhancement descriptions that make it look weird, but not seeing the tabs when in editor.

**Comment by RaphaelDDL** [comment]
@kmalone86 my dumb VSCode formatter sometimes wraps my pasted objects into a weird `()`

Gotta figure out why. But template fixed.

File: hero/aither.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+            "awakenUpgrade": false,
+            "cooldown": 2,
+            "name": "Guard",
+            "soulAcquire": 1,
+            "description": "Recovers the ally's Health with water energy. Amount recovered is proportional to Attack.",
+            "enhancement": [{
+                "description": "+5% healing",
+                "resources": [{ "item": "molagora", "qty": 1 }, { "item": "gold", "qty": 4000 }]
+            },{
+               	"description": "+5% healing",
```
Comment (kmalone86) [inline hero/aither.json]:
```suggestion
                "description": "+5% healing",
```

File: hero/aither.json [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+                "resources": [{ "item": "molagora", "qty": 3 }, { "item": "mysterious-flash", "qty": 2 }, { "item": "gold", "qty": 22000 }]
+            },{
+               	"description": "+10% healing",
```
Comment (kmalone86) [inline hero/aither.json]:
```suggestion
                "description": "+10% healing",
```

**Comment by Triharder-E7** [comment]
@kmalone86 I went ahead and approved the commits, so hopefully that fixes all the spacing issues. I hadn't realized tabs were such a trap! I'll use the new hero template from now on, and hopefully that will keep things pretty looking.

**Comment by kmalone86** [comment]
Everyone has their preference with tabbing vs spacing, but looks like github has a set way of displaying, which will look weird if not consistent.

**Pull request merged**